synapse/config/homeserver.py

```diff
@@ -92,5 +92,4 @@ class HomeServerConfig(RootConfig):
         TracerConfig,
         WorkerConfig,
         RedisConfig,
-        FederationConfig,
```

Comment:
I was confused by this, but turns out it is duplicated above (on line 61).

Reply:
sorry yes, should have clarified that!